.github/workflows/test-contracts.yml

```diff
@@ -47,11 +47,11 @@ jobs:
         working-directory: ./contracts
 
       - name: Run tests
-        run: forge test -vvv
+        run: forge test --no-match-path script/**/*.sol -vvv
```

Comment:
should we also capture this exclusion regex in the `package.json`'s test command definition? also what's the rationale for not running the script tests as part of CI?

Reply:
The test I wrote to test the deployment is meant to be ran after a deployment by a verifier (which may not be the deployer!), ingesting environment variables input by the verifier. This would obviously fail in CI without contracts already deployed. And yes, I guess it would need to be excluded in package.json too.

Maybe there is a better way to do this, but I think in general there should be no need to put tests that are in the script/ folder into CI. If we put tests there meant for CI, they should be in the test/ folder.

Reply:
I guess by default, I could have the test run after running the deployment script using mock values. But then the test would not fail if the correct environment variables are not supplied, which seems like a way that the verifier might make a mistake.

Reply:
+1 to having it as a part of a script and not a test

Reply:
https://github.com/Layr-Labs/eigenda/pull/1415/commits/e1e14498b97626ef09b9b39e2d222a10713a8ffa